### PR TITLE
PLAT-144492 Update for kitchen-sink dependencies.

### PR DESCRIPTION
**Pull request by alexandrumorariu**
I updated the kitchen-sink to work with the newest versions of React, Prop-Types, Enact and Agate. I also removed the Sandstone dependence.

### CODE REVIEW

File: kitchen-sink/src/index.js [hunk 1]
```diff
@@ -1,10 +1,9 @@
-import React from 'react';
 import {render} from 'react-dom';
 import App from './App';
 
 const appElement = (
 	<App
-		skin="cobalt"
+		skin="gallium"
 		accent="#FC7982"
```
Comment (stanca-pop-lgp) [inline kitchen-sink/src/index.js]: If you changed the skin to Gallium, please also update the accent and highlight colors.

Comment (alexandrumorariu) [reply kitchen-sink/src/index.js]: Done

File: kitchen-sink/src/views/MainPanel.js [hunk 2]
```diff
@@ -32,27 +30,24 @@ const MainPanel = kind({
...
 				<Cell shrink>
 					<Button>Click me</Button>
-					<Button type="standard" icon="home" />
-					<ToggleButton
-						underline
-						type="standard"
-						toggleOffLabel="Off"
-						toggleOnLabel="On"
-					/>
+					<Button icon="plus" type="standard">Home</Button>
```
Comment (stanca-pop-lgp) [inline kitchen-sink/src/views/MainPanel.js]: Wasn't this supposed to be an icon button? I mean just a button with a "home" icon.

Comment (alexandrumorariu) [reply kitchen-sink/src/views/MainPanel.js]: Done

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Review by jeonghee27 — APPROVED** [review]
LGTM

**Pull request merged**